sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprContainerIT.java

```diff
@@ -59,9 +65,10 @@ public class DaprContainerIT {
   private static final String KEY = "my-key";
   private static final String PUBSUB_NAME = "pubsub";
   private static final String PUBSUB_TOPIC_NAME = "topic";
+  private static final String APP_FOUND_MESSAGE_PATTERN = ".*application discovered on port 8081.*";
 
   @Container
-  private static final DaprContainer DAPR_CONTAINER = new DaprContainer("daprio/daprd")
+  private static final DaprContainer DAPR_CONTAINER = new DaprContainer("daprio/daprd:1.14.4")
```

Comment:
This is better since it makes the test not break because of a bug in runtime.

Reply:
Ideally, this is coming from a centralized configuration that is updated once in the repository.

Reply:
This has been fixed in a previous PR. The image tag for Dapr Container has been extracted to a constant.